exampleSite/static/admin/config.yml

```diff
@@ -307,23 +307,6 @@ collections:
         widget: "object"
         fields:
           - {label: "Discord Name", name: "discord_id", widget: "string"}
-  # encounter metadata
```

Comment:
I don't think we need this anymore. We are not using anything from the `data` folder correct?

Reply:
Yeah. I suppose I had ideas about this before, but you are absolutely correct that data is probably too much overhead for something like this.